pythonforandroid/build.py

```diff
@@ -408,8 +410,28 @@ def prepare_build_environment(self, user_sdk_dir, user_ndk_dir,
                 toolchain_prefix))
             ok = False
 
+        # Get toolchain-specific folders:
         self.toolchain_prefix = toolchain_prefix
         self.toolchain_version = toolchain_version
+        self.toolchain_path = (self.ndk_dir + "/toolchains/" +
```

Comment:
See comment below. Essentially, `stdarg.h` is missing somehow from the regular includes provided by CrystaX and hidden away in this toolchain-specific gcc include folder. I don't know why, or why this issue doesn't appear for other recipes that use `-nostdinc` and include `stdio.h` - if you have an idea what to investigate, I'll happily look into it. For now, this seemed like the cleanest solution to avoid this issue and also allow other recipes to use this path if necessary